tests/more-tests/numeric_subscripts/test_subscript_1d_arrays.mdl

```diff
@@ -0,0 +1,93 @@
+{UTF-8}
+Inflow A[One Dimensional Subscript]=
+	Rate A[One Dimensional Subscript]
+	~
+	~		|
+
+Stock A[One Dimensional Subscript]= INTEG (
+	Inflow A[One Dimensional Subscript],
+		0)
+	~
```

Comment:
If this test-model can be integrated UI would prefer to add it to the test-models repo with the canonical output, as most of the test.
I would keep in this folder only the non-valid vensim/XMILE models that cannot be integrated so makes no sense to translate (e.g. undefined variables)

Reply:
Done!